.travis.yml

```diff
@@ -15,7 +15,6 @@ cache:
   ccache: true
   directories:
     - $HOME/cache
-
 env:
```

Comment:
13863 our travis.yml likely needs some more changes...

Reply:
should be fixed up now